app/export.go

```
@@ -207,17 +209,15 @@ func (app *IrisApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs [
+			panic("expected validator, not found")
 		}
 
 		validator.UnbondingHeight = 0
 		if applyAllowedAddrs && !allowedAddrsMap[addr.String()] {
 			validator.Jailed = true
 		}
 
-		if err := app.StakingKeeper.SetValidator(ctx, validator); err != nil {
-			panic(err)
-		}
+		app.StakingKeeper.SetValidator(ctx, validator)
```

golangci-lint (1.62.2)

Error return value of `app.StakingKeeper.SetValidator` is not checked

(errcheck)